roles/openshift_node_upgrade/tasks/restart.yml

```diff
@@ -22,7 +22,6 @@
     - "{{ openshift.common.service_type }}-master-controllers"
     - "{{ openshift.common.service_type }}-node"
   failed_when: false
-  when: openshift.common.is_containerized | bool
```

Comment:
openvswitch and openshift-node are stopped first thing in the role, then containerized master and etcd services are stopped only if containerized.